smart-contracts/tasks/bridge.ts

```diff
@@ -37,23 +39,50 @@ task('bridge:send', 'Send tokens to a pool across a relay bridge')
         )
       }
 
+      if (!asset) {
+        const assetName = await new Select({
+          choices: ['native', ...Object.keys(net.assets)],
+          message: 'Please choose the asset you want to bridge:',
+          name: 'asset',
+        }).run()
+        if (assetName === 'native') {
+          asset = rawEthers.ZeroAddress
```

Comment:
we could replace all mentions of `rawEthers` by `ethers` in the script

Reply:
Ha! But then that would break zksync, no?